server/website/website/tasks/periodic_tasks.py

```diff
@@ -82,8 +82,6 @@ def run_background_tasks():
             # Check that there are enough results in the workload
             LOG.info("Not enough results in workload %s (# results: %s, # required: %s).",
                      workload_name, num_wkld_results, MIN_WORKLOAD_RESULTS_COUNT)
-            workload.status = WorkloadStatusType.PROCESSED
```

Comment:
Why do we no longer need to update the workload status to PROCESSED? If we don't then won't the workload status still be set to 'PROCESSING' after this background task finishes?

Reply:
Same.